CMakeLists.txt

```
@@ -0,0 +1,136 @@
+cmake_minimum_required(VERSION 3.0)
```

```suggestion
cmake_minimum_required(VERSION 3.9)
```
LTO support requires newer CMake.